rest/auth.py

```diff
@@ -66,6 +67,14 @@
             print("invalid value '%s' for guests parameter, guests will be "
                   "disabled." % (guests_allowed_s))
 
+    if settings.has_option('roles', 'trusted_sources'):
```

Comment:
In the documentation above, only trusted_sources_allowed is described. Should it also describe the trusted_sources parameter?

Reply:
trusted_sources_allowed is a mistake in the documentation, only trusted_sources works.